front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -163,8 +167,11 @@ export function AgentMessage({
     switch (message.status) {
       case "succeeded":
       case "failed":
-      case "cancelled":
         return message;
+      case "cancelled":
+        return streamedAgentMessage.status === "created"
+          ? { ...streamedAgentMessage, status: "cancelled" }
```

Comment:
@spolu I added a commit to change that -> we indeed want streamedAgentMessage but still status cancelled 🙏🏻

Reply:
LGTM